app/controllers/admin/rankings_controller.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+class Admin::RankingsController < ApplicationController
+  include AdminHelper
+  before_action :require_admin
+
+  def index
+    categories = %w[food drink lt]
+    @rankings = {}
+
+    categories.each do |category|
+      @rankings[category] = {
+        single: Vote.ranking_by_single(category),
+        total: Vote.ranking_by_total_votes(category)
+      }
```

Comment:
配列の中身を繰り返して新しい何か（今回ではhash）を作るといった操作をする場合rubyは専用のメソッドがあるのでそちらを使いましょう。

https://railsguides.jp/active_support_core_extensions.html#index-with `index_with`使えます。

```suggestion
    @rankings = categories.index_with do |category|
      {
        single: Vote.ranking_by_single(category),
        total: Vote.ranking_by_total_votes(category)
      }
```